docs/configuration/tracing.md

```diff
@@ -98,3 +98,41 @@ Træfik supports two backends: Jaeger and Zipkin.
     #
     id128Bit = true
 ```
+
+## DataDog
+
+```toml
+# Tracing definition
+[tracing]
+  # Backend name used to send tracing data
+  #
+  # Default: "jaeger"
+  #
+  backend = "datadog"
+
+  # Service name used in Zipkin backend
```

Comment:
Could you replace `Service name used in Zipkin backend` by `Service name used in DataDog backend`